src/ui/profile_preferences.ui

```diff
@@ -3391,6 +3391,19 @@ you can use it but there could be issues with aligning columns of text</string>
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QCheckBox" name="checkBox_askTlsAvailable">
+         <property name="text">
+          <string>Allow secure connection reminder</string>
```

Comment:
I'd a tooltip to this for those who see this setting without any context to better explain what this option does.

Reply:
<img width="322" alt="Screen Shot 2022-11-12 at 2 53 11 PM" src="https://user-images.githubusercontent.com/3058178/201492239-ec398f8f-ecc1-4fad-82d9-215b66d40796.png">